pupil-spa/src/app/connectivity-check/connectivity-check.component.html

```diff
@@ -10,5 +10,14 @@
       </div>
     </div>
   </div>
+  <div *ngIf="isUnsupportedBrowser === true" class="grid-row">
+    <header>
+      <h1 class="heading-xlarge">This broswer is not supported</h1>
+    </header>
+    <p class="lede" tabindex="0">
+      You can view additional information on supported browsers
+      in the MTC technical guidance on GOV.UK: <a href="https://www.gov.uk/government/collections/multiplication-tables-check">https://www.gov.uk/government/collections/multiplication-tables-check</a>
```

Comment:
Whilst 'technical guidance' may be the approved copy, the guidance appears to be in Annex  A of the 'System Guidance' and if the reader does manage to find it, it says:

> 'The MTC has been designed to support all common browsers. However, please note
> certain optional features, such as the question reader, will not work on Internet Explorer
> or Edge due to the technical limitations of these browsers.'

I'd suggest changing 'MTC technical guidance' to 'MTC system guidance' which is a real thing.

Reply:
Also, Edge does support the Speech Synthesis API and the latest Chromium builds will be fine.